perllib/FixMyStreet/Cobrand/HighwaysEngland.pm

```diff
@@ -223,4 +306,75 @@ sub dashboard_export_problems_add_columns {
     });
 }
 
+# select distinct category from contacts where category ilike '%litter%' or category ilike '%clean%' or category ilike '%fly%tip%';
+# search to find categories in all contacts and then manually edited
+sub _cleaning_categories { [
+    'Accumulated Litter',
+    'Cleanliness Issue',
+    'Cleanliness Sub Standard',
+    'Cleansing',
+    'Excessive or dangerous littering',
+    'Fly Tipping on a road, footway, verge or open space',
+    'Fly Tipping',
+    'Fly tipping',
+    'Fly-Tipping',
+    'Fly-tipping',
+    'Flytipping and dumped rubbish',
+    'Flytipping',
+    'Flytipping/flyposting',
+    'General (Cleanliness)',
+    'General Litter / Rubbish Collection',
+    'General fly tipping',
+    'Hazardous fly tipping',
+    'Litter On Road/Street Cleaning',
+    'Litter and Bins',
+    'Litter in Parks & Open spaces',
+    'Litter in the street',
+    'Litter removal',
+    'Litter',
+    'Littering',
+    'Littering and cleanliness',
+    'Rubbish or fly tipping on the roads',
+    'Street Cleaning',
+    'Street Cleansing',
+    'Street cleaning and litter',
+    'Street cleaning',
+    'Street cleansing',
+    'Sweeping & Cleansing Hazard',
+    #Bench/cycle rack/litter bin/planter,
```

Comment:
Can all these commented out entries be removed?

Reply:
I thought it was nice to keep the list in case we have to make any changes in the near future.